docs/security/cors.rst

```diff
@@ -1,8 +1,355 @@
-.. include:: /stub-topic.txt
+Enabling Cross-Origin Requests (CORS)
+=====================================
 
-|stub-icon| Enabling CORS
-=========================
+By `Mike Wasson <https://github.com/MikeWasson>`_ | Originally Published: May 25, 2015
 
-.. include:: /stub-notice.txt
+What is CORS?
+-------------
+
+Browser security prevents a web page from making AJAX requests to another domain. This restriction is called the *same-origin policy*, and prevents a malicious site from reading sensitive data from another site. However, sometimes you might want to let other sites make cross-origin requests to your web app; for example, to call a REST API. 
+
+`Cross Origin Resource Sharing <http://www.w3.org/TR/cors/>`_ (CORS) is a W3C standard that allows a server to relax the same-origin policy. Using CORS, a server can explicitly allow some cross-origin requests while rejecting others. CORS is safer and more flexible than earlier techniques such as `JSONP <http://en.wikipedia.org/wiki/JSONP>`_. This topic shows how to enable CORS in your ASP.NET 5 application (including MVC 6).
```

Comment:
I would like to maintain a clean split between docs for ASP.NET 5 and MVC 6 docs. Please move the MVC 6 specific content to the MVC TOC as part of https://github.com/aspnet/Docs/issues/136 and then cross link the two articles.